.github/ISSUE_TEMPLATE/code_formatting_guidelines_specific_issue.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Formatting or code style guidelines issue
+about: Discuss editor / tooling formatting concerns and code style guidelines
+title: ''
+labels: [style-guide, under-discussion]
+assignees: '@nojaf,@KathleenDollard,@dsyme'
+---
+
+Depending the concern being discussed, erase the other top sections and review/tick what applies the most.
+
+# Is your request about a feature in code formatting (using fantomas, .editorconfig, editors with formatting options)?
+
+Formatting infrastructure decision checklist:
+* [ ] If this is code formatter discussion, please suggest a detailed list of settings names and their default values, with their behaviour
+* [ ] the choice for tooling support should be based on ease of implementation in code formatter
+* [ ] the choice for tooling output should be based on
+  * [ ] conserving original formatting but for specific predicate that makes code significantly less maintainable
+  * [ ] maximizing uniformity of a specific construct across a particular codebase
+* [ ] I would like to contribute testing the feature implementation as end user
+* [ ] I would like to contribute unit tests so the tooling implementation covers my use cases
+* [ ] I would like to contribute implementation
+  * [ ] pro-bono
+  * [ ] sponsored
+  * [ ] sponsoring
+* [ ] The implementation requires the several options upfront (rather than just the most "agreed upon") [^1]
+* [ ] This is something that is common place in other langauges (add items nested under this list with one language per line)
+* [ ] There are examples of codeformatters settings in other languages / tooling available (add items nested under this list with one link per line), add screenshots to the description
+* [ ] There are examples of codeformatters implementation available (add items nested under this list with one link per line)
+
+[^1] this is generally evaluated by the person implementing and the entity potentially sponsoring the work
```

Comment:
Once more, we Fantomas maintainers strongly object to additional settings! Merely proposing a PR doesn't grant you authority here. Modifying settings in Fantomas should only be considered as a final option, subject to approval by the maintainers.